_posts/2025-06-27-a2a-project-lanunches-java-sdk.adoc

```diff
@@ -0,0 +1,127 @@
+---
+layout: post
+title: 'A2A Project launches Java SDK'
+date: 2025-06-27
+tags: release
+synopsis: 'We’re thrilled to announce the launch of the A2A Java SDK, made possible by WildFly and Quarkus teams’ contribution.'
+author: maeste
+---
+
+== *A2A Project Launches Java SDK*
+
+The agent revolution just took a massive leap forward! Following the
+recent landmark https://developers.googleblog.com/en/google-cloud-donates-a2a-to-linux-foundation/[announcement that Google has donated the Agent2Agent
+(A2A) protocol to the Linux Foundation], we’re thrilled to announce the
+launch of the *https://github.com/a2aproject/a2a-java[A2A Java SDK]*, made possible by WildFly and Quarkus 
+teams’ contribution.
+
+
+=== *A New Era Under Linux Foundation Stewardship*
+
+The A2A protocol’s transition to the Linux Foundation represents more
+than just a change of governance: it’s a commitment to vendor-neutral,
+community-driven innovation that ensures this critical interoperability
+standard remains open and accessible to all. With more than 100
+companies now supporting the protocol, we’re witnessing the formation of
+what industry leaders are calling "`an open, interoperable Internet of
+Agents.`"
+
+=== *Why Java SDK Matters*
+
+Here’s where things get exciting from a technical perspective: *true
+polyglot agent ecosystems*.
+
+The agent landscape has been fragmented, with Python dominating AI/ML
+workflows, JavaScript powering web-based agents, and Java serving as the
+backbone of enterprise backend systems. Each language ecosystem operated
+in isolation, creating invisible walls that limited the collaborative
+potential of multi-agent systems.
+
+Our Java SDK shatters these barriers by implementing the A2A protocol
+specification natively in Java, enabling:
+
+* *Enterprise-grade agent integration* with existing Java
+infrastructure +
+* *Seamless interoperability* between Java agents and those written in
+Python, JavaScript, or any A2A-compatible language +
+* *Production-ready deployments* leveraging Java’s mature ecosystem and
+well-tested enterprise capabilities
+
+Want to see this interoperability in action? Explore our
+https://github.com/a2aproject/a2a-samples/tree/main/samples/multi_language/python_and_java_multiagent[multi-language
+sample implementation&#44;] which demonstrates how Python and Java
+agents collaborate seamlessly.
+
+=== *Technical Excellence: The Mutiny-Zero Advantage*
+
+The Java SDK leverages *mutiny-zero* as its reactive foundation, a
```

Comment:
correct me if im wrong - is the sdk reactive all the way ? or is it just parts of it that are reactive?

It would be good to at least highlight reactive streams is not enforced on you as a user - but available if/when you need it?